assets/js/components/Button.js

```diff
@@ -83,7 +83,10 @@ Button.displayName = 'Button';
 
 Button.propTypes = {
 	onClick: PropTypes.func,
-	children: PropTypes.string.isRequired,
+	children: PropTypes.oneOfType( [
+		PropTypes.arrayOf( PropTypes.node ),
+		PropTypes.node,
+	] ),
```

Comment:
Isn't `PropTypes.node` sufficient for `children`? I thought `node` described anything that could be rendered.

Reply:
This is correct, silly me.